### PR TITLE
Use cloud function to clear redis cache

### PR DESCRIPTION
**Pull request by shifucun**
Remove the hacky way of clearing cache from website UI and use Cloud Function to trigger it.

### CODE REVIEW

**Review by beets — APPROVED** [review]
this is nice, thanks bo!

cc/ @chejennifer

File: tools/clearcache/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+This tool clears all the [redis memcache](https://pantheon.corp.google.com/memorystore/redis/locations/us-central1/instances/webserver-cache/details?project=datcom-website-prod) of prod website deployment and it's
+deployed as a cloud function. Much of the steps are borrowed from [here](https://cloud.google.com/memorystore/docs/redis/connect-redis-instance-functions#python).
+
+## Deploy the function to Cloud Function
+
+```bash
+gcloud config set project datcom-website-prod
+
+gcloud functions deploy clear_cache \
```
[the remark this answers is not in the document]
Comment (shifucun) [reply tools/clearcache/README.md:13]: This is only needed when main.py is changed. Added to the doc.

**Comment by shifucun** [comment]
Thanks for review!

**Comment by pradh** [comment]
@shifucun  This is nice!  Could we also update the redis cache clearing section in go/dcrelease?  (sorry if you already have a CL out)

**Comment by shifucun** [comment]
> @shifucun This is nice! Could we also update the redis cache clearing section in go/dcrelease? (sorry if you already have a CL out)

Yep, I have updated that

**Pull request merged**